### ISSUE TITLE
Add Simulator-Release build configuration in bar-descriptor template

### ISSUE THREAD

**Issue by guarnold** [blackberry/VSPlugin#438]
The user can select the Release configuration in the Simulator platform in the Configuration Manager. However, we don't have this build configuration in the bar-descriptor assets configuration. As a consequence, the app cannot be deployed.

**Comment by dbrgss** [comment]
Included in v1.01.0040

**Comment by zadmarbella** [comment]
failed verification on v1.01.0040

**Comment by guarnold** [comment]
Configuration Simulator-Release will be included automatically when a new project / bar-descriptor is created.

However, for existing projects, the following code must be included manually in the bar-descriptor.xml file, by using the XML editor (substitute the 2 PROJECT_NAME by the correct project name).

``` xml
  <configuration name="Simulator-Release">
    <platformArchitecture>x86</platformArchitecture>
    <asset path="Simulator-Release/PROJECT_NAME" entry="true" type="Qnx/Elf">PROJECT_NAME</asset>
  </configuration>
```

If user does not include the above code and use Simulator-Release configuration, he might experience the following error:
BBNATIVEPACKAGER : error : Attribute Entry-Point: file cannot be found in the list of packaged files: PROJECT_NAME

**Comment by guarnold** [comment]
Fix available in build v1.01.0040-2

**Comment by zadmarbella** [comment]
verified 1.01.0042 
win 7 2010
win 8 2012
user is able to create a new bar-descriptor from Configuration manager and it include Simulator-Release configuration in bar-descriptor code.
